Protactor/specs/jankiness.js

```diff
@@ -1,8 +1,13 @@
 const assert = require('assert')
 
 describe('should test the jankiness', () => {
+    beforeAll(() => {
+        browser.waitForAngularEnabled(false)
+    })
+
     it('tests jankiness with non optimized behavior', async () => {
         browser.get('https://googlechrome.github.io/devtools-samples/jank/')
+        await browser.sleep(5000)
```

Comment:
is 5s sleep really required?

Reply:
So occasionally it's not able to find the element that it clicks right after, so this is to try to mitigate that.  Perhaps because of the way the site works the element may not load immediately even after the page loaded?  I reduced it down to 1s.